GliaWidgets/Sources/Component/Header/Header.swift

```diff
@@ -60,7 +60,7 @@ final class Header: BaseView {
     }
 
     func renderProps() {
-        backButton?.isEnabled = props.backButton != nil
+        backButton?.isHidden = props.backButton == nil
         if let backButtonProps = props.backButton {
             backButton?.props = backButtonProps
         }
```

Comment:
I guess here you check if `props.backButton` is there before hiding `backButton` and hide it in the `else` block. Both ways works though :)